extensions/amp-dynamic-css-classes/0.1/amp-dynamic-css-classes.js

```diff
@@ -91,8 +91,8 @@ function normalizedReferrers(win) {
  * @param {!Array<string>} classes
  */
 function addDynamicCssClasses(win, classes) {
-  const documentElement = win.document.documentElement;
-  const classList = documentElement.classList;
+  const body = win.document.documentElement.body;
```

Comment:
So, this would be just a tiny bit more complicated here. We need to wait for `body` to be actually available. See `waitForBody` in `dom.js`.